sdk/typescript/src/types/transactions.ts

```diff
@@ -107,6 +107,39 @@ export type OwnedObjectRef = {
   reference: SuiObjectRef;
 };
 
+export type DevInspectResults = {
+  effects: TransactionEffects;
+  results: DevInspectResultsType;
+};
+
+export type ResultType = 'Ok' | 'Err;';
```

Comment:
@666lcz is this the right way to handle Rust `Result` here?

Reply:
Result will deserialize to `{'OK': [(),()]}` and `{'Err': ""}`. The best way to figure out the schema is to write a ts e2e test